org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/parser/CompSym.java

```diff
@@ -1,7 +1,7 @@
 
 //----------------------------------------------------
 // The following code was generated by CUP v0.11a beta 20060608
```

Comment:
why do we have this file checked in...

Reply:
removed this file and other generated files. The `.gitingore` file is also updated.